packages/shared/user-data/GraphQL/config.ts

```diff
@@ -80,9 +80,10 @@ export const config = {
     label: "Enable backend processing routines",
     legacyKey: "devtools.features.enableRoutines",
   },
-  backend_keepAllTraces: {
+  backend_sampleAllTraces: {
     defaultValue: Boolean(false),
     internalOnly: Boolean(true),
+    label: "Keep all Honeycomb events",
     legacyKey: "devtools.features.keepAllTraces",
```

Comment:
I guess it isn't _quite_ what the high-risk flag was added for but this setting should probably be flagged as high-risk (since we don't want to it left on in most cases)?

```js
  highRisk: Boolean(true),
```

Reply:
This is a case where i'd like to be able to leave it on for a bit and see what is happening on the backend

Reply:
Isn't that kind of circumstantial? (Would you want more people turning this on and leaving it on?)

The run book you shared says:
> Please do not keep this setting on more than you have to. We almost always operate right at the limit of our honeycomb budget and if we keep all internal traces we will go over our Honeycomb limits and start dropping valuable data. 🙏

To me, that sounds like a reasonable use of the high-risk flag UI.